tpgtools/main.go

```diff
@@ -257,7 +256,7 @@ func loadDocument(packagePath string, f *os.FileInfo) *openapi.Document {
 	}
 	err = yaml.Unmarshal(b, document)
 	if err != nil {
-		glog.Exit(err)
+		panic(err)
```

Comment:
`glog.Exit` is generally better than panic I believe? Why would we change this

Reply:
oh this a debugging artifact, will remove

Reply:
done